.github/CODEOWNERS

```
@@ -35,7 +35,7 @@ web/gui/ @jacekkolasa
 
 # Ownership by filetype (overwrites ownership by directory)
 *.am @Ferroin @tkatsoulas
-*.md @tkatsoulas
+*.m?(d|dx) @tkatsoulas @andrewm4894
```

```suggestion
*.md @tkatsoulas @andrewm4894
*.mdx @tkatsoulas @andrewm4894
```

Sorry, for some stupid reason, I was under the impression this file supports extended syntax